solr/CHANGES.txt

```diff
@@ -106,6 +106,8 @@ Other Changes
 
 * SOLR-17285: SolrJ RemoteSolrException moved to SolrClient. (@samuelrivascoding)
 
+* SOLR-17321: Bump up minimum Java version for Apache Solr is now 21, and for SolrJ, it is 17. (Sanjay Dutt, David Smiley)
```

Comment:
```suggestion
* SOLR-17321: Minimum Java version for Apache Solr is now 21, and for SolrJ, it is 17. (Sanjay Dutt, David Smiley)
```

Reply:
Actually this ticket not meant to upgrade Java. It's more of experiment with Jetty upgrade. But I have not merge the Java upgrade ticket yet, so I am going to make this change there. (# 2682)